Feature hover conact modal

pylyps opened the pull request.

pylyps committed:
Feature hover shadow

Added feature hover - shadow

pylyps committed:
feature-modal

Added modal window

Fredy228 commented on src/partials/modal-window-franchasie.html:
```diff
@@ -0,0 +1,58 @@
+<div class="backdrop is-hidden" data-modal>
+  <div class="modal">
```
Измени класс, вместо "modal" на "franchise"
И везде ниже тоже 
У нас будут три модалки, если назвать все modal, то будут конфликт

pylyps replied:
Изменил. Везде кроме btn--close- кнопка пусть выглядит везде однородно.

Fredy228 commented on src/partials/modal-window-franchasie.html:
```diff
@@ -0,0 +1,58 @@
+      <legend class="modal__fieldset">
+        Заполните форму заявки ниже, и мы Вам перезвоним
+      </legend>
+      <label class="=">
```
Исправь название класса "=", на нормальное название)

pylyps replied:
Done

Fredy228 commented on src/partials/modal-window-franchasie.html:
```diff
@@ -0,0 +1,58 @@
+<div class="backdrop is-hidden" data-modal>
+  <div class="modal">
+    <fieldset>
```
Убрать тег "fieldset" и обернуть всю форму в тег "form"

pylyps replied:
Done, хотя я бы оставил филдсет, т.к. буде заголовок формы. Ну то мелочи :)

Fredy228 commented on src/partials/modal-window-franchasie.html:
```diff
@@ -0,0 +1,58 @@
+<div class="backdrop is-hidden" data-modal>
+  <div class="modal">
+    <fieldset>
+      <legend class="modal__fieldset">
```
убрать тег" legend", использовать тег "strong"

pylyps replied:
Поменял, хотя опять же непонятно почему - филдсет используется для заголовкоов форм.

Fredy228 reviewed (changes requested):
Чтобы подключить scss файл, в папке "components" есть файл "index.scss", туда подключаются все файлы в этой папке

pylyps committed:
Bugfixing_modal_window

1. Changed classes,
2. Asigned modal-window-franchase to index.scss
3. Class with modal was left only for <class="modal__btn-close">
4. Changed fielset to label with comments in Githab
